runtime/doc/change.txt

```diff
@@ -914,9 +914,10 @@ inside of strings can change!  Also see 'softtabstop' option. >
 
 							*Y*
 ["x]Y			yank [count] lines [into register x] (synonym for
-			yy, |linewise|).  If you like "Y" to work from the
-			cursor to the end of line (which is more logical,
-			but not Vi-compatible) use ":map Y y$".
+			yy, |linewise|).
+			{Nvim} By default, Y is mapped to yank text from the
+			cursor to the end of line, which is more logical, but
+			not vi-compatible (see |Y-eol|).
```

Comment:
I don't think we need to mention vi compatibility, given what's been done everywhere else.